lib/shared/utils.js

```diff
@@ -30,50 +30,9 @@
 
 'use strict';
 
-const traverse = require('traverse');
+const searchConfig = require('../../configuration/fhir_service.search');
```

Comment:
@richardcbrown I'm not sure that require files outside `lib` folder is good idea (but can't say that it's something bad in the same time). Initially, I thought that files in `lib` folder should require the stuff inside `lib` folder only. I prefer qewd-up functions call lib functions and not vice versa

Here is another problem, we have 2 config places (`configuration` and `lib/config`) and it can confuse the developer who will work with the code. `configuration` is the part of QEWD-up architecture, `config` is part of my concept. TBH, I'd like to merge them to once but not sure how to do it the best way

For now, let's leave it as is, I'd like you to note about this weak moment

Reply:
The only idea I have for it right now, is to
1. Attach `searchConfig` in `onWorkerLoad` to have access to it via `ExecutionContext`. Please note that you should add mock version of searchConfig to `spec/support` folder and update worker in mocks
2. create a new prop `searchConfig` in `ExecutionContext` (the same idea like `globalConfig` prop) to expose search config
3. In `resourceService.fetchResources` pass `searchConfig` to `mapQuery` as param

It makes the concept more clean (from my point of view ofc), however, it doesn't resolve a problem of 2 config places.